lib/utils.js

```diff
@@ -64,7 +64,11 @@ function getHookValue(data, job, key) {
 }
 
 function makeHook(format, job) {
-  format = format || DEFAULT_FORMAT
+  try {
+    format = JSON.parse(format)
```

Comment:
This no longer allows a string, since `JSON.parse(str)` throws an error since it's not valid JSON.